### PR TITLE
Import jira issues

### PR DESCRIPTION
**Pull request by ralfhandl**
Dummy PR, do not merge: should only contain temporary JSON files with issues.

- [x] Deferred issues
  - [x] use label "wontfix" 
- [x] Closed issues
  - [x] derive label "wontfix" from `fields.resolution?.name`
  - [x] 1-100
  - [x] 101-200
  - [x] 201-300
  - [x] 301-400
  - [x] 401-500
  - [x] 501-600
  - [x] 601-700
  - [x] 701-800
  - [x] 801-900
  - [x] 901-1000
  - [x] 1001-1100
  - [x] 1101-1200
  - [x] 1201-1300
  - [x] 1301-1400
  - [x] 1401-1482

### CODE REVIEW

File: lib/transformJiraIssues.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "mikep@microsoft.com": "mikepizzo",
+  "ralf.handl@sap.com": "ralfhandl",
+  "stefan@drees.name ": "sthagen",
+};
+
+const gitHubIssues = [];
+
+for (const j of jiraIssues) {
+  const g = {
+    title: j.fields.summary,
```
Comment (HeikoTheissen) [inline lib/transformJiraIssues.js]:
```suggestion
    title: `${j.key}: ${j.fields.summary}`,
```
Have the old Jira key in the title

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

File: lib/transformJiraIssues.js (hunk 1 above)
Comment (ralfhandl) [reply lib/transformJiraIssues.js]: I did not want the old/migrated issues to stick out, so I only attached a backlink to the end of the description/body.

Comment (ralfhandl) [reply lib/transformJiraIssues.js]: We could add a label "from Jira" to the migrated issues so we can easily find them via issue filters, and remove the ODATA-xxx prefix.

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — COMMENTED** [review]
Suggestion: Automatically add a label like `core/4.02` based on the `fixVersions`.

**Comment by ralfhandl** [comment]
> Suggestion: Automatically add a label like `core/4.02` based on the `fixVersions`.

We already add labels for `fixVersions`:

image

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Pull request closed without merging**